Restructuring docs folder (#204)
kyma-project/website: docs/content-modification/write-blog-posts.md

# Writing blog posts

Follow these steps to add a new post on the website:

1. Create a new folder in the `content/blog-posts` folder with a name following the `{DATE}-{ADDRESS}` format.
2. Create in created dir file with `index.md` name.
3. Prepare the content in created markdown file.
4. Make a pull request.
5. Wait for the review.
6. After the merge, the site rebuilds automatically and the blog post appears.

The blog post should have the following format:

``` yaml
---
title: {TITLE}
author: 
  name: {AUTHOR}
tags:
  - {TAG_1}
  - {TAG_2}
  - {TAG_3}
  ...
---

{CONTENT}
```

Replace these parameters with real values:

- `{DATE}` is your blog post's publication date, in the `YYYY-MM-DD` format.

> - `{ADDRESS}` is the last part of the site's address in the location bar. For example, if you want your blog post to appear under `https://kyma-project.io/blog/{DATE}/some-address`, this var should contains `some-address`.

Author: `https://kyma-project.io/blog/2019/02/14/release-notes-07`

- `{AUTHOR}` is your name with an optional position name.
- `{CONTENT}` is written in Markdown and/or HTML. The content must include the `<!-- overview -->` comment which indicates the part of the blog post that will be displayed on the main page. The **Read more** button appears at the end of the paragraph.

>**NOTE:** The `<!-- overview -->` comment cannot be placed at the beginning of the blog post. In such a case, the **Read more** button appears straight after the title of the blog post and the content itself is not displayed.

Example:

``` yaml
---
title: "Latin's not dead"
author: 
  name: "Cicero, Roman Philosopher"
tags:
  - Lorem
  - Ipsum
  - Dolor
  - Sit
  - Amet
---

Lorem ipsum dolor sit amet, consectetur adipiscing elit. 

<!-- overview -->

Integer volutpat interdum eros a malesuada. Proin porttitor, leo eu dignissim posuere, ante nibh aliquam ipsum, pharetra pharetra nunc libero eu massa.
```

---

## Determine the type of blog post

> By writing a post you can determine what type it is. By specifying the type, the content of the post is automatically adapted to the template of the type. Specifying the type is optional - no type information will display the post in the default way.

Author: Without any buttons etc, only raw content from blog post

You can also use HTML tags to show and position images:

``` HTML
<img src="{RELATIVE_PATH_OR_LINK}" >
```

See the example usage with mixed elements:

``` HTML
<p align="center">
    <img src="./logo.png" width="235">
</p>
```

<p align="center">
    <img src="../assets/logo.png" width="235">
</p>

---

## Link between files

### Link to blog posts

Use this pattern to add links to other blog posts:

``` Markdown
[Another blog post](/blog/{DATE}/{ADDRESS})
```
where: 
- `{DATE}` is blog post's publication date, in the `YYYY/MM/DD` format.
- `{ADDRESS}` is the last part of the site's address in the location bar.

Example:

```
[Introduction](/blog/2018/07-24/introduction-project-kyma)
```

### Link to documentation

Use this pattern to add links to other documents:

``` Markdown
[Overview - In a nutshell](/docs/{VERSION}/{DOCS_TYPE}/{DOC_TOPIC}#{HASH})
```
where: 
- `{VERSION}` is version of pointed documentation.
- `{DOCS_TYPE}` is type of documentation (at the moment only available are `root` and `components`).
- `{DOC_TOPIC}` is topic type of pointed documentation.

> - `{HASH}` is hash of pointed documentation (this is optionally field).

Reviewer: ```suggestion
- `{HASH}` is an optional anchor link to the referenced documentation.
```
